nmostesting/NMOSTesting.py

```diff
@@ -561,7 +598,7 @@ def run_tests(test, endpoints, test_selection=["all"]):
         elif test == "IS-09-02":
             # This test has an unusual constructor as it requires a system api instance
             test_obj = test_def["class"](apis, SYSTEMS, DNS_SERVER)
-        elif test == "IS-04-04" or test == "IS-05-03":
+        elif test in ["IS-04-04", "IS-05-03", "BCP-006-01-02"]:

```

Comment:
@jonathan-r-thorpe if all/most Controller test suites will follow the same form, would it make sense to use `isinstance(test_def["class"], ControllerTest)` here?

Reply:
Yes, that is a much better/more maintainable approach.